bundles/org.openhab.binding.solarman/src/main/java/org/openhab/binding/solarman/internal/state/LoggerState.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.solarman.internal.state;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * @author Catalin Sanda - Initial contribution
+ */
+@NonNullByDefault
+public class LoggerState {
```

Comment:
See other comment about communication between Thing handler and remote controller. I don't think this class is needed and it can be simplified.

Reply:
Removed the class.